services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/filter_column.tsx

```diff
@@ -98,6 +108,18 @@ const FilterColumn = ({
     />)
   }
 
+  if ([EVIDENCE_BETA1, EVIDENCE_BETA2].includes(flagset)) {
```

Comment:
How about making a constant in flagOptions for this grouping? If we add EVIDENCE_BETA3 later on, this line may be missed.